package.json

```diff
@@ -49,7 +49,7 @@
     "@types/hosted-git-info": "^3.0.2",
     "@types/jest": "^26.0.24",
     "@typescript-eslint/eslint-plugin": "^4.28.3",
-    "@zeit/ncc": "^0.22.3",
+    "@vercel/ncc": "^0.28.6",
```

Comment:
[`@vercel/ncc@0.29.0`](https://github.com/vercel/ncc/releases/tag/0.29.0) outputs ESM, so it does not work on Actions... 😭